woof-distro/x86/void/voidpup32/DISTRO_PET_REPOS

```diff
@@ -0,0 +1,63 @@
+#------------------
+#PKG_DOCS_PET_REPOS - where to download the pet pkgs databases from.
+#------------------
+# 1|2|3
+#   1 - domain. for testing the url.
+#   2 - full URI of the database file.
+#   3 - name of db file when local and after being processed into standard format
+#  (in the case of PET databases, the names are the same and no processing is required)
+
+if ( echo "$BUILD_FROM_WOOF" | grep -q ';' ) ; then
+	WCE_BRANCH=$(echo "$BUILD_FROM_WOOF" | cut -f 1 -d ';')
+else
+	WCE_BRANCH=testing
+fi
+
+PKG_DOCS_PET_REPOS="
+z|http://smokey01.com/peebee/voidpup/Packages-puppy-vpup32-extra|z
+z|https://raw.githubusercontent.com/puppylinux-woof-CE/woof-CE/testing/woof-distro/x86/slackware/14.2/Packages-puppy-slacko14.2-official|z
```

Comment:
Is VoidPup using packages built against Slackware 14.2? I'm surprised if this works 🙀